evergreen/eg_app/views.py

```diff
@@ -108,62 +108,45 @@ def addCookies(response: HttpResponse, cookies: dict[str, str]) -> HttpResponse:
     return response
 
 
-def validate(request):
-    if request.method == "POST":
-
-        password = request.POST.get("password")
-        email = request.POST.get("email")
-
-        valid_pass = True
-        valid_email = True
-
-        if not val.validate_password(password):
-            valid_pass = False
+@csrf_exempt
```

Comment:
Was already there before me, this is git just tracking things oddly. It is necessary because a person cannot have their csrf token if they have not even registered yet. This is the problem of having the same main page be the signin/up.

Reply:
In that case we should probably make separate signup/login pages. Django has built-in tools to make that fairly simple

https://learndjango.com/tutorials/django-login-and-logout-tutorial#log-in-page
https://www.pythontutorial.net/django-tutorial/django-loginview/

(Though both of these also use csrf tokens, so I'm not sure that's actually the problem?)

Reply:
We can do this later.

Reply:
Never mind I think that somehow snuck in when I merged and didn't see it.